CODEOWNERS

```diff
@@ -5,7 +5,7 @@
 # For more details, read the following article on GitHub: https://help.github.com/articles/about-codeowners/.
 
 # These are the default owners for the whole content of the `examples` repository. The default owners are automatically added as reviewers when you open a pull request, unless different owners are specified in the file.
-* @a-thaler @clebs @suleymanakbas91 @lilitgh @rakesh-garimella @hisarbalik
+* @a-thaler @lilitgh @rakesh-garimella @hisarbalik
```

Comment:
Could you please add @skhalash @chrkl @lindnerby as well

Reply:
I somehow can't edit the file further. I created the PR by editing the file on GitHub and the branch is protected.
If we merge this I can open a new one and add them.